.gitignore

```diff
@@ -31,3 +31,4 @@ npm_licenses.tar.bz2
 
 # Ignore parser debug
 y.output
+.vscode
```

Comment:
Please add this to your own local gitignore. We cannot add all special files of all IDEs in the world here.